### PR TITLE
add smartBCH

### PR DESCRIPTION
**Pull request by StM4H4**
add smartBCH, Chain ID: 10000.

Fixes # .

Changes proposed in this pull request:
- Add smartBCH to network.

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -646,5 +646,15 @@
...
+  },
+  "10000": {
+    "key": "10000",
+    "name": "smartBCH",
+    "shortName": "BCH",
+    "chainId": 10000,
+    "network": "mainnet",
+    "multicall": "0x1b38EBAd553f218e2962Cb1C0539Abb5d6A37774",
+    "rpc": ["https://smartbch.greyh.at/"],
+    "explorer": "https://www.smartscan.cash/"
```
Comment (ChaituVR) [inline src/networks.json:658]:
```suggestion
    "explorer": "https://www.smartscan.cash"
```

**Comment by ChaituVR** [comment]
Looks good@StM4H4 but you also need to add network logo image here 
https://github.com/snapshot-labs/snapshot.js/tree/master/src/networks

**Comment by StM4H4** [comment]
@ChaituVR Thank you for reviewing, sir. But I can not upload files in that folder. It shows:
> Uploads are disabled.
> File uploads require push access to this repository.
 Am I doing something wrong?
Can I attach the URL of the image directly?
[](https://s2.coinmarketcap.com/static/img/coins/64x64/1831.png)

**Comment by ChaituVR** [comment]
> @ChaituVR Thank you for reviewing, sir. But I can not upload files in that folder. It shows:
> 
> > Uploads are disabled.
> > File uploads require push access to this repository.
> 
> Am I doing something wrong? Can I attach the URL of the image directly?

@StM4H4 You can upload image to this branch on your fork :)

**Comment by StM4H4** [comment]
I got it!
Thank you, sir. @ChaituVR

**Review by bonustrack — APPROVED** [review]

**Pull request merged**